test-harness/infratests/unit.go

```diff
@@ -20,7 +17,8 @@ import (
 	terraformCore "github.com/hashicorp/terraform/terraform"
 )
 
-type ResourceAttributeValueMapping map[string]map[string]string
+type AttributeValueMapping map[string]string
```

Comment:
remind me again why we're only supporting string comparison for our plan reconciliation check?

Reply:
Everything in terraform 11 is flattened to a string in the plan. List indexes become part if the attribute key, as an example